app/src/types/product.type.ts

```diff
@@ -0,0 +1,5 @@
+export type Product = {
+  image?: string;
+  description: string;
```

Comment:
Maybe we should change description to name so that in future we can add a longer description?